docs/gzip.external.md

```diff
@@ -1,14 +1,14 @@
 GZIP is used to compress text based responses before sending them to the client who will uncompress them.
 
-This allows clients to download a considerable smaller file by taking a small expense of compressing on the server when sending the response.
+This allows clients to download a considerably smaller file by taking a small expense of compressing on the server when sending the response.
 
-By general GZIP can save your users 80%+ on file sizes to download, in most cases.
+In general GZIP can save your users 80%+ on file sizes to download.
 
-This issue is related to a external asset that was used which does not have GZIP enabled and could save your users a minimum of 20kb or more from the size of files needed to download to display your page.
+This issue is related to a external asset that was used which does not have GZIP enabled and could save your users a minimum of 20KB or more from the size of files needed to download to display your page.
 
-We advice contacting the remote vendor and showing the information as to the benifits that will be gained by enabling GZIP at [passmarked.com/performance/inspect/enable-gzip-to-decrease-size-that-the-user-has-to-download-from-local-site](http://passmarked.com/performance/inspect/enable-gzip-to-decrease-size-that-the-user-has-to-download-from-local-site) which will also guide as to how this can be enabled.
+We advise contacting the remote vendor and showing the information as to the benefits that will be gained by enabling GZIP at [passmarked.com/performance/inspect/enable-gzip-to-decrease-size-that-the-user-has-to-download-from-local-site](http://passmarked.com/performance/inspect/enable-gzip-to-decrease-size-that-the-user-has-to-download-from-local-site). This will also guide you as to how this can be enabled.
```

Comment:
Link does not appear to work anymore, new link is https://passmarked.com/library/performance/network/gzip.internal/enable-gzip-to-decrease-the-size-of-assets-that-the-user-has-to-download-from-local-site